### ISSUE TITLE
Questions about data set partitioning

### ISSUE THREAD

**Issue by sanguiyh** [VulCNN/VulCNN#17]
![image](https://github.com/CGCL-codes/VulCNN/assets/51226130/2f35359a-464b-492d-93d8-40b41f2f498b)

In the code, Kflod cross-validation divides the data set. Each time the code is divided into test.pkl and train.pkl, will it cause all the data to be used as the training set? Won't the result be too high?
Kfold cross-validation should divide the data set into a training set and a test set. The training set is divided into k parts. During each cross, one part will be used as the validation set, and the rest will be used as the training set. The test set will never be used. That's right for training. The code randomly divides the complete data set into k parts, causing each test data to be trained.

**Comment by jinmingzhong** [comment]
![image](https://github.com/CGCL-codes/VulCNN/assets/86002161/e2526665-076b-4a49-81b4-4527b59c3e0a) In 5-fold cross-validation, _**classifier**_ is initialized for each validation.

**Comment by sanguiyh** [comment]
Got it, thank you very much

**Comment by sanguiyh** [comment]
> ![image](https://user-images.githubusercontent.com/86002161/277144940-e2526665-076b-4a49-81b4-4527b59c3e0a.png) In 5-fold cross-validation, _**classifier**_ is initialized for each validation.

Got it, thank you very much
